src/main/java/com/bettercloud/vault/api/Logical.java

```diff
@@ -51,20 +51,32 @@ public Logical(final VaultConfig config) {
      * @throws VaultException If any errors occurs with the REST request (e.g. non-200 status code, invalid JSON payload, etc), and the maximum number of retries is exceeded.
      */
     public LogicalResponse read(final String path) throws VaultException {
+        return read(path, 0);
```

Comment:
Please specify why we are passing 0 here. Possibly with proper static final variable name